packages/library/src/core/timeline/items/audio.ts

```diff
@@ -91,7 +91,7 @@ const defaultPayload = {
   pan: 0,
   rampUp: 0,
   rampDown: 0,
-  panningModel: 'equalpower',
+  panningModel: 'equalpower' as PanningModelType,
```

Comment:
This was needed to prevent a build error. It seems unrelated to this PR, but I included it anyway